guiprep.pl

```diff
@@ -231,170 +235,178 @@
 my $main = MainWindow->new(
 	-title =>		'Guiprep Pre-processing Toolkit - '.$currentver,
 );
+my $int = $main->interp;
 
 # Set window size (pixels) the script TRIES to open with.
 $main->geometry($geometry);
 
 # Set the minimum window size.
-$main->minsize(qw(600 480));
+$main->minsize(qw(1024 768));
 
-# Set an interesting palette color combo.
-$main->setPalette("$palette");
+# Set an interesting theme color combo.
+$main->ttkSetTheme("$theme");
 
-my $mainbframe = $main->Frame()->pack(-anchor => 'nw');
+my $mainbframe = $main->ttkFrame()->pack(-anchor => 'nw');
 
 my $bottomlabel = $mainbframe->Label(
 	-text => "Working from the $pwd directory.",
-	-font => '-*-Helvetica-Medium-R-Normal--*-160-*-*-*-*-*-*'
 )->pack(-anchor => 'nw');
 
-my $maintopframe = $main->Frame(
+my $maintopframe = $main->ttkFrame(
 )->pack(-anchor => 'n', -expand => 'y', -fill => 'both');
 
-my $book = $maintopframe->NoteBook( -ipadx => 6, -ipady => 6);
-
-my $page2 = $book->add("page2", -label => "Select Options", -raisecmd => sub {$interrupt = 1;chdir $pwd;});
-my $page1 = $book->add("page1", -label => "Process Text", -raisecmd => \&updateblist);
-my $page8 = $book->add("page8", -label => "Search", -raisecmd => sub{$interrupt = 1; searchclear(); $filesearchindex1 = 1;
-									@searchfilelist=(); $main->update; $main->Busy; searchincdec(); $main->Unbusy;});
-my $page3 = $book->add("page3", -label => "Headers & Footers", -raisecmd => sub {emptybox(); $interrupt = 1;chdir $pwd;});
-my $page4 = $book->add("page4", -label => "Change Directory", -raisecmd => sub {$interrupt = 1;chdir $pwd;});
-my $page6 = $book->add("page6", -label => "Program Prefs", -raisecmd => sub {$interrupt = 1;chdir $pwd;});
-my $page5 = $book->add("page5", -label => "About", -raisecmd => sub {$interrupt = 1;chdir $pwd;});
+my $book = $maintopframe->ttkNotebook();# -ipadx => 6, -ipady => 6);
+
+$main->bind('<<NotebookTabChanged>>',
+	sub{
+		for ($book->tab($book->select())->[7]) {
```

Comment:
This block looks like a spaces/tabs mixup?

Reply:
On a personal level, I'm team spaces, but my understanding was this whole file was tab-based.
<img width="975" alt="Screenshot 2023-01-19 at 9 48 02 AM" src="https://user-images.githubusercontent.com/5185341/213473026-7e094712-b074-415b-bc8b-06031712846e.png">

Do you just mean that I put in two tabs after the for when it should have only been one? I can definitely fix the indentation. Sorry.

Reply:
I'm team spaces as well but let's be consistent in the file. this code segment in the PR just looks totally wonky:
<img width="918" alt="Screen Shot 2023-01-20 at 10 17 21 AM" src="https://user-images.githubusercontent.com/5853628/213776373-bbfbd1a8-792c-4974-a8b2-ccf541527791.png">

Reply:
Fixed (I think?) with ad76a9b.